ElementaryTasks/Arrays.cs

```diff
@@ -49,5 +49,109 @@ public void SumOfElements()
             }
             Console.WriteLine(sum);
         }
-    }   
+
+        public void MostCommonValue()
+        {
+            int[] numbers = { 11, 10, 12, 13, 14, 13, 16, 13, 15, 4 };
+            int longestOccurance = 0;
+            int mostCommonValue = 0;
+
+            for (int i = 0; i < numbers.Length; i++)
+            {
+                int counter = 0;
+
+                for (int j = 0; j < numbers.Length; j++)
+                {
+                    if (numbers[j] == numbers[i])
+                    {
+                        counter++;
+                    }
+                }
+
+                if (counter > longestOccurance)
+                {
+                    longestOccurance = counter;
+                    mostCommonValue = numbers[i];
+                }
+            }
+            Console.WriteLine(mostCommonValue);
```

Comment:
Please add for each method some conclusion.
instead of `Console.WriteLine(mostCommonValue);`
use somethind like
`Console.WriteLine($"The most common value in array is {mostCommonValue}");`

Please add it here and everywhere

Reply:
Changed here and will be use this rule for next tasks